backend/routers/userRoutes.js

```diff
@@ -43,8 +43,12 @@ router.patch(
     authController.updatePassword
 );
 router.get("/me", authController.protect, userController.getMe);
-
-router.use(authController.protect);
+// update
+router.patch(
```

Comment:
Put?

Reply:
should be 'patch' it is action for updating partial resource